projects/ngqp/core/src/lib/accessors/util.ts

```diff
@@ -0,0 +1,51 @@
+import { ControlValueAccessor } from '@angular/forms';
+import { DefaultControlValueAccessorDirective } from './default-control-value-accessor.directive';
+import { NGQP_BUILT_IN_ACCESSORS } from './ngqp-accessors';
+
+/**
```

Comment:
I know this is how it was before, but can we add `@internal` here to prevent it from being publicly documented?

Reply:
Added.